c-example/example.c

```diff
@@ -0,0 +1,50 @@
+#include <stdio.h>
+
+#include "spglib.h"
+
+int main(void) {
+    SpglibDataset *dataset;
+    int i, j, size;
+    const char *wl = "abcdefghijklmnopqrstuvwxyz";
+
+    // Wurtzite structure (P6_3mc)
+    double lattice[3][3] = {
+        {3.111, -1.5555, 0}, {0, 2.6942050311733885, 0}, {0, 0, 4.988}};
+    double position[4][3] = {
+        {1.0 / 3, 2.0 / 3, 0.0},
+        {2.0 / 3, 1.0 / 3, 0.5},
+        {1.0 / 3, 2.0 / 3, 0.6181},
+        {2.0 / 3, 1.0 / 3, 0.1181},
+    };
+    int types[4] = {1, 1, 2, 2};
+    int num_atom = 4;
+
+    // SplibDataset has to be freed after use.
+    dataset = spg_get_dataset(lattice, position, types, num_atom, 1e-5);
+
+    printf("International: %s (%d)\n", dataset->international_symbol,
```

Comment:
Can we add Wikipedia references in the comments to some of these? It's fine only in the C  example because it will be the main entry-point.

Reply:
I don't see what you mean about wikipedia references...

Reply:
Like what's the reference to `International symobl` and so on.

Reply:
If so, we should refer International Tables for Crystallography or Bilbao crystallographic server because wikipedia may be misleading.

Reply:
Of course, whichever sources can explain it to new-commers.

Reply:
OK, it is done at 1baad19.